src/opm/parser/eclipse/share/keywords/keyword_list.cmake

```diff
@@ -9,6 +9,25 @@
 # you are therefore encouraged to go to the opm-simulators repository and update
 # the file: opm/autodiff/MissingFeatures.cpp
 
+#Some keywords are found to be of 'special' structure:
+#W             WHAT
+#ECONT         Terminates with two /
+#COALNUM       matrix structure
+#COMPDATM      alias for COMPDATL
+#COMPINJK      undef default
+#COMPRP        strange defaults
```

Comment:
The strange defaults in `COMPRP` are "OK"; no default can be assigned in json, and the C++ code will have to look like this:
```
if item.defaultApplied(0)
    value = do_some_fancy_calculation_to_get_a_usable_value();
```

this is OK - we have many such keywords.